lib/zstd_stats.h

```diff
@@ -0,0 +1,164 @@
+/*
+    zstd - standard compression library
```

Comment:
I fully agree.
So, how to combine 1. and 2. ?
What do you think of having a `lib/debug` directory ?